fastlane/devices.txt

```diff
@@ -0,0 +1,3 @@
+Device ID	Device Name	Device Platform
```

Comment:
What came to my mind just yesterday: Is there a way to _not_ expose our devices publicly?

Reply:
We could add the `devices.txt` to the `.gitignore` afaik the devices will not be un-registered if you remove them anyways, so you could add your own devices to the file. use `bundle exec fastlane ios register_devices` but not push the file. This would add your device to the developer portal but not touch any existing devices.